.travis.yml

```
@@ -82,7 +82,7 @@ after_success:
 
   - |
     # upload coverage
-    if [[ "$ZDS_TEST_JOB" != "none" ]]; then
+    if [[ "$ZDS_TEST_JOB" != "none" && "$ZDS_TEST_JOB" != "*ci_turbo*" ]]; then
```

```suggestion
    if [[ "$ZDS_TEST_JOB" != "none" && "$ZDS_TEST_JOB" != *"ci_turbo"* ]]; then
```

Ce n'est pas plutôt ça ? Si j'en crois le reste du code.

---

bien vu